demos/message-monitor-view.js

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2022, Tony Atkins
+ * Copyright 2011-2020, Colin Clark
+ *
+ *  Licensed under the MIT license, see LICENSE for details.
+ */
+
+/*global ArrayBuffer, DataView, CodeMirror*/
+(function (fluid) {
+    "use strict";
+
+    /**
+     * Logs incoming MIDI messages from all <code>youme.connection</code>s, globally.
+     */
+    fluid.defaults("youme.messageMonitorView", {
+        gradeNames: "fluid.codeMirror",
+
+        codeMirrorOptions: {
+            lineWrapping: true,
+            readOnly: true
+        },
+
+        theme: "flockingcm",
+        lineNumbers: true,
+        lineWrapping: true,
+        readOnly: true,
+
+        distributeOptions: {
+            // TODO: This is probably, umm, a bit heavy-handed.
+            target: "{/ youme.connection}.options",
```

Comment:
It's not bad, but I'd put a namespace in here so that these definitions can be located/deduplicated - also you may wish to parcel the listener into a grade rather than writing it out longhand which may create subtle expansion issues longterm

Reply:
This grade hasn't been rewritten yet, I just straight up copied it over.  But I will keep it in mind when I get there.

Reply:
I wrote a replacement demo and removed all the (now) unused functions.